.github/workflows/zombie-bite-common.yml

```diff
@@ -348,9 +348,9 @@ jobs:
         run: |
           COL_PORT=$(jq -r .collator_port ${ZOMBIE_BITE_BASE_PATH}/ports.json)
```

Comment:
We should move this to a `just` call to easily run locally. I can change it in this branch if you want.

Reply:
Sure, please do 👍